src/default/assets/js/src/typedoc/components/Search.ts

```diff
@@ -154,7 +154,7 @@ namespace typedoc.search
         }
 
         for (var i = 0, c = Math.min(10, res.length); i < c; i++) {
-            var row = data.rows[res[i].ref as any as number];
+            var row = data.rows[+res[i].ref];
```

Comment:
Are you just casting? In that case, could we do that more explicitly?
```suggestion
            var row = data.rows[Number(res[i].ref)];
```